impeller/renderer/backend/gles/capabilities_gles.cc

```diff
@@ -127,6 +127,10 @@ CapabilitiesGLES::CapabilitiesGLES(const ProcTableGLES& gl) {
     GLint value = 0;
     gl.GetIntegerv(GL_MAX_SAMPLES_EXT, &value);
     supports_offscreen_msaa_ = value >= 4;
+  } else if (desc->GetGlVersion().major_version >= 3 && desc->IsES()) {
+    GLint value = 0;
+    gl.GetIntegerv(GL_MAX_SAMPLES_EXT, &value);
```

Comment:
I don't think you can safely query this without first verifying that `GL_EXT_framebuffer_multisample` is present.

Reply:
I believe this is core in 3.0:

https://registry.khronos.org/OpenGL-Refpages/es3.0/html/glRenderbufferStorageMultisample.xhtml

Reply:
I don't see `GL_MAX_SAMPLES_EXT` mentioned there.  The only place I see it mentioned is in documentation for that extension (ex: https://www.khronos.org/opengl/wiki/GL_EXT_framebuffer_multisample).  I'm pretty sure it's not standard because it has the `_EXT` postfix.